source/includes/fact-valid-namespace-remap.rst

```diff
@@ -0,0 +1,33 @@
+
+The following restrictions apply to namespace remapping:
```

Comment:
Is it possible to add this limitation:

```
You can't specify a namespace remap and set the ``reversible`` flag to 
  ``true``.
```

If not, no worries. We can do this in a follow up ticket after the release.

Reply:
Just to make sure we're not adding too much content that isn't reviewed by the rest of the writing team, could you put this in a fast-follow ticket?